testdata/time-equal.go

```diff
@@ -0,0 +1,27 @@
+package pkg
```

Comment:
After configuring `revive` to only apply `time-equal` rule, when doing
```
./revive -config defaults.toml -formatter friendly testdata/time-equal.go
```
No failure is found.
This might be due to (silent) type checking errors.

Reply:
interesting, it works for me :thinking: 😬